nemo/collections/nlp/modules/common/megatron/clip_grads.py

```diff
@@ -24,6 +25,7 @@
     from apex.multi_tensor_apply import multi_tensor_applier
     from apex.transformer import parallel_state
     from apex.transformer.tensor_parallel.layers import param_is_not_tensor_parallel_duplicate
+    from apex.contrib.optimizers.distributed_fused_adam import DistributedFusedAdam
 
```

Comment:
If it's a contrib in a very recent version of apex, best to put this under it's own try catch block with its own flag for testing